### PR TITLE
Release 3.1.0 Changelog

### PR DESCRIPTION
**Pull request by avilaton**
Created from the current diff from 3.0.0 to develop, needs cleanup.

--
addresses #799

### CODE REVIEW

**Comment by avilaton** [comment]
Perhaps I should arrange changes by their topic like i18n, docs, etc

**Comment by jirikuncar** [comment]
@avilaton can you keep the format of the release notes similar to the past ones? Sometimes it is also necessary to go through the details in the commit messages. Lastly, we could start grouping by: Added/Fixed/Removed keywords.

**Comment by adeadman** [comment]
Is there any update on when this will be released? I'm still blocked by #681 which means my application has an open redirect vulnerability while pinned on the 1.7.x version, so getting this released would be a big help.

File: CHANGES [hunk 1]
```diff
@@ -3,6 +3,53 @@ Flask-Security Changelog
 
 Here you can see the full list of changes between each Flask-Security release.
 
+Version 3.1.0
+-------------
+
+Released TBD
+
+- 8919828 i18n: added Japanese translation
```
Comment (jirikuncar) [inline CHANGES]: @avilaton thanks! Can you please match the style with the rest of change log?

Comment (avilaton) [reply CHANGES]: I'm on it, I have a question, why was `next`removed here https://github.com/mattupstate/flask-security/commit/79545a92a1ebb601386fb34820508058ad7cdff6

**Comment by avilaton** [comment]
@jirikuncar I split the changelog into added, fixed, removed the best I could. I linked all changes to the corresponding PR number and noted down the contributor's handle for acknowledgement, please let me know if you think this is ok or if you think I should remove them.

**Comment by avilaton** [comment]
Perhaps I could help you so we can publish updates more often so there isn't such a large overhead for a release.

**Review by jirikuncar — APPROVED** [review]

**Comment by jirikuncar** [comment]
@avilaton thanks! I hope the things will start moving quicker soon. Meantime, please have a look at #822.

**Comment by avilaton** [comment]
Thanks to you!

**Pull request merged**